server/views/pages/licence/partials/hdcCurfewTimes.njk

```diff
@@ -1,25 +1,28 @@
 {% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
 
```

Comment:
This was using `curfewTimes` as the parameter name which was confusing me so changed this to `hdcLicenceData`